cmd/podman/containers/run.go

```diff
@@ -125,6 +126,12 @@ func run(cmd *cobra.Command, args []string) error {
 	if err := createInit(cmd); err != nil {
 		return err
 	}
+	var fd uint
+	for fd = 3; fd < 3+runOpts.PreserveFDs; fd++ {
+		_, err := unix.FcntlInt(uintptr(fd), unix.F_GETFL, 0); err != nil && err != syscall.Errno(0) {
+			return err
```

Comment:
I think this should be a wrapped error to tell the user which FD is invalid.

Reply:
The FcntlInt code needs to be in a util_linux.go branch.
Does podman-remote throw an error when you do --preserver-fds?

Reply:
Added utils_linux.go  `--preserver-fds` is disabled in remote.

Reply:
I think you want to have similar code in exec.go and perhaps attach.go as well?